src/components/ActionDetail/FormControl.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+MIT License
+
+Copyright (c) 2021-present, Elastic NV
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+import { EuiFormRow, EuiFieldText } from "@elastic/eui";
+import React, { ChangeEventHandler } from "react";
+
+export function FormControl({
+  label,
+  name,
+  noPrepend,
+  onChange,
+  value,
+}: {
+  label?: string;
+  name: string;
+  noPrepend?: boolean;
+  onChange: ChangeEventHandler<HTMLInputElement>;
+  value: string;
+}) {
+  return (
+    <EuiFormRow label={label ?? "Command"}>
+      <EuiFieldText
+        onChange={onChange}
+        prepend={noPrepend === true ? undefined : name}
```

Comment:
We're in 100% agreement about the nuanced implications of `undefined` vs `null`. Unfortunately `prepend` must be a react string or `undefined` and triggers a TS error instead. We'd need to merge an upstream update to EUI to change this. I think I probably tried to make it `null` when I wrote this code even and decided to simply use `undefined` rather than resist the API of this component.